packages/nocodb/src/db/BaseModelSqlv2/relation-data-fetcher.ts

```
@@ -0,0 +1,1557 @@
+              is_group: true,
+              logical_op: 'and',
+            }),
+          ],
+          query,
+        );
+
+        return (
+          await baseModel.execAndParse(query, null, { raw: true, first: true })
+        )?.count;
+      } catch (e) {
+        throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)